centos_ci.sh

```diff
@@ -305,7 +305,8 @@ function build_and_test() {
   cd $GOPATH/src/github.com/minishift/minishift
   make prerelease synopsis_docs link_check_docs
   # Run integration test with 'kvm' driver
-  MINISHIFT_VM_DRIVER=kvm make integration GODOG_OPTS="-tags=basic,openshift,version,cmd-config,profile -format pretty"
+  # TODO: Remove it. Added for test purpose
```

Comment:
Don't forget ;)

Reply:
Through make integration_all will included three more tags ```cmd-addons, flags and proxy``` except the default ```basic,openshift,version,cmd-config,profile```. Not sure why it was not added before to ci.

Reply:
Probably, when it was added, we didn't update `centos_ci.sh` script.